Lib/fontTools/ttLib/tables/otTables.py

```diff
@@ -536,7 +536,7 @@ def preWrite(self, font):
 		cov.glyphs = [ item[1] for item in items]
 		alternates = []
 		setList = [ item[-1] for item in items]
-		for  set in setList:
+		for set in setList:
```

Comment:
Absolutely. I think this code predates the `set()` builtin, though... Feel free to improve!